### DISCUSSION TITLE
perilib status?

### DISCUSSION THREAD

**Discussion by mark-hahn** [jrowberg/i2cdevlib#640] [General]
I see from the perilib repos that it never got off the ground.  But I'm confused.  Why are there zero issues on those repos and no mention of perilib status here?

**Comment by jrowberg** [comment]
It's not quite that it never got off the ground; a great deal of working foundational code for Arduino and Python in particular has been written and tested pretty well. However, there aren't any device libraries yet, which is where most of the end-user (developer) value will come from. I've had to focus elsewhere since I started. It has nearly zero visibility compared to i2cdevlib. There's a dead-simple example that actually gets raw data from the MPU-6050 here:

https://github.com/perilib/perilib-arduino-core/blob/master/examples/Register_TWI_MPU6050_direct/Register_TWI_MPU6050_direct.ino

There are more than a few possible next steps on the project, but my life needs to calm down first. :)
